test/mox_test.exs

```diff
@@ -392,40 +407,71 @@ defmodule MoxTest do
       set_mox_private()
 
       verify!(CalcMock)
+      verify!(SciCalcOnlyMock)
       expect(CalcMock, :add, fn x, y -> x + y end)
+      expect(SciCalcOnlyMock, :exponent, fn x, y -> x * y end)
 
-      message = ~r"expected CalcMock.add/2 to be invoked once but it was invoked 0 times"
-      assert_raise Mox.VerificationError, message, &verify!/0
+      error = assert_raise(Mox.VerificationError, fn -> verify!(CalcMock) end)
+      assert error.message =~ ~r"expected CalcMock.add/2 to be invoked once but it was invoked 0 times"
+      refute error.message =~ ~r"expected SciCalcOnlyMock.exponent/2"
```

Comment:
**Annotation:** Even though the previous tests are in the `describe "verify!/1"` section, they used to be calling the `verify!/0` method! With this change now we're actually :one: calling the `verify!/1` method and :two: asserting that the behavior changes appropriately when there are multiple mocks used within the same test case.